app/selenium_ui/confluence/pages/pages.py

```diff
@@ -83,6 +85,20 @@ def wait_for_comment_field(self):
     def click_edit(self):
         self.wait_until_clickable(PageLocators.edit_page_button).click()
 
+    def wait_for_resources_loaded(self):
+        sleep_time = 0.1
+        attempts = 30
+        total_sleep = 0
+
+        for _ in range(attempts):
+            loaded = self.execute_js("return require('confluence-editor-loader/editor-loader').resourcesLoaded();")
+            if loaded:
+                return
+            time.sleep(sleep_time)
+            total_sleep += sleep_time
+            print(f'Total sleep time: {total_sleep}')
+        print('confluence-editor-loader resource could not be loaded')
```

Comment:
This error message is confusing. Also, do we want to continue test even if resource was not loaded?
Let's change to something like:
`print(f'WARNING: confluence-editor-loader resource was not loaded in {timeout} seconds.')`